installer/README.md

```diff
@@ -0,0 +1,49 @@
+# deno_install
+
+- This command installs executable deno script.
+
+## Features
+
+- Install executable script into ~/.deno/bin
+
+## Usage
+
+```sh
+$ deno_install https://deno.land/std/http/file_server.ts
+> file_server requests network access. Grant permanently? [yN] # Grant the permissions to use command.
+> y
+> Successfully installed file_server.
+$ file_server # now you can use installed command!
+```
+
+## Requirements
+
+- wget #TODO: Remove
+
+## Installing
+
+### 1. Install deno_install
+
+deno_install can be installed by using itself.
+
+```sh
+deno -A https://deno.land/std/install/deno_install.ts https://deno.land/std/install/deno_install.ts
+```
+
+### 2. Add `~/.deno/bin` to PATH
+
+```
+echo 'export PATH="$HOME/.deno/bin:$PATH"' >> ~/.bashrc # change this to your shell
+```
+
+## Create Executable Script
+
+- Add shebang to top of your deno script.
+  - This defines what permissions are needed.
+
+```sh
+#!/usr/bin/env deno --allow-read --allow-write --allow-env --allow-run
```

Comment:
It's pretty cool, but... not all scripts have shebang line - then it `deno_install` should handle manually specified perm flags as described by @ry:

>So now people can run this:

> `deno_install file_server https://deno.land/std/http/file_server.ts --allow-write --allow-net`

> This will create a new executable shell script called $PATH/file_server, which would have the following contents:

> #!/bin/sh
> deno run --allow-write --allow-net https://deno.land/std/http/file_server.ts $@

If we skip this feature in first version we can drop `wget` as well